Source/Addon/Support Files/Help/HTML Help Source/Main.html

```diff
@@ -69,6 +69,6 @@ <h3>Disable the Device</h3>
 <h3>Add a Bitfile</h3>
 <p>To add a bitfile, select the file from the <strong>Bitfile path</strong>.</p>
 <h3>Set the RIO Address</h3>
-<p>Enter in the target FPGA address (usually RIO#) in <strong>RIO Device Address</strong>.</p>
+<p>Enter in the target FPGA address (RIO#, or an alias) in <strong>RIO Device Address</strong>.</p>
```

Comment:
How about this:
"Enter the target FPGA address in **RIO Device Address**. The address can be either the RIO number or an alias."